stream_alert_cli/test.py

```diff
@@ -76,19 +75,21 @@ def __init__(self, context, config, print_output):
         # Create the RuleProcessor. Passing a mocked context object with fake
         # values and False for suppressing sending of alerts to alert processor
         self.processor = StreamAlert(context, False)
+        self.cli_config = config
         # Use a list of status_messages to store pass/fail/warning info
         self.status_messages = []
         self.total_tests = 0
         self.all_tests_passed = True
         self.print_output = print_output
         helpers.setup_mock_firehose_delivery_streams(config)
 
-    def test_processor(self, filter_rules, validate_only=False):
+    def test_processor(self, rules_filter, files_filter, validate_only):
         """Perform integration tests for the 'rule' Lambda function
 
         Args:
-            filter_rules (list|None): Specific rule names (or None) to restrict
-                testing to. This is passed in from the CLI using the --rules option.
+            rules_filter (set): A collection of rules to filter on, passed in by the user
+            filter_rules (set): Specific rule names to restrict testing to. This is
```

Comment:
shouldn't this be named `files_filter`

Reply:
Yeah I think so - this must have gotten mixed up. Fixing